src/test/java/com/google/javascript/gents/singleTests/goog_scope_local_alias.js

```diff
@@ -0,0 +1,29 @@
+goog.provide('a.b.c.Base1');
+goog.provide('a.b.c.Base2');
+goog.provide('a.b.c.Base3');
+goog.provide('a.b.c.Base4');
+
+goog.scope(function() {
+var b = a.b;
+/** @constructor */
+b.c.Base1 = function() {}
+});
+
+goog.scope(function() {
+var c = a.b.c;
+/** @constructor */
+c.Base2 = function() {}
+});
+
+goog.scope(function() {
+/** @constructor */
+a.b.c.Base3 = function() {}
+});
+
+goog.scope(function() {
+var c = a.b.c;
+var foo = 1;
```

Comment:
I'm guessing this is intended to verify that `foo` is not exported.   Instead of naming the var `foo`, can you use a name that describes what its intended goal is?